integreat_cms/cms/templates/pois/poi_form.html

```diff
@@ -156,8 +156,9 @@ <h3 class="heading font-bold text-black">
                         </div>
                         <div>
                             <label>{% trans 'Position' %}</label>
-                            <div class="help-text">
-                                {% trans 'If you leave these fields blank, they are automatically derived from the address.' %}
+                            <div>
+                                <input type="checkbox" id="auto-fill-coordinates">
+                                <label for="auto-fill-coordinates">{% trans 'Calculate coordinates automatically from address' %}</label>
```

Comment:
```suggestion
                                <label for="auto-fill-coordinates" class="secondary !inline">
                                    {% trans 'Calculate coordinates automatically from address' %}
                                </label>
```
I would also suggest to make the label smaller and define it as inline element so it aligns next to the checkbox instead of breaking into the next line.